### PR TITLE
RET-2058: ET3 Notification

### PR DESCRIPTION
**Pull request by robin-ieong**
https://tools.hmcts.net/jira/browse/RET-2058

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by jsinghm13 — DISMISSED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 1]
```diff
@@ -12851,6 +12881,426 @@
...
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3SelectAllThatApply",
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3OtherTypeDocumentName",
+    "UserRole": "caseworker-employment",
```
Comment (Itay-Kirsh) [inline definitions/json/AuthorisationCaseField.json:12947]: R only user role

**Review by Itay-Kirsh — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -12851,6 +12881,426 @@
...
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3SendDocByFirstClass",
+    "UserRole": "caseworker-employment-api",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3ClaimantAndRespondentAddresses",
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "CRUD"
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:13098]: R permission

File: definitions/json/AuthorisationCaseField.json [hunk 3]
```diff
@@ -12851,6 +12881,426 @@
...
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3ClaimantAndRespondentAddresses",
+    "UserRole": "caseworker-employment-englandwales",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3ClaimantAndRespondentAddresses",
+    "UserRole": "caseworker-employment",
+    "CRUD": "CRUD"
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:13116]: R permission

File: definitions/json/AuthorisationCaseField.json [hunk 4]
```diff
@@ -12851,6 +12881,426 @@
...
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3ClaimantAndRespondentAddressesLabel",
+    "UserRole": "caseworker-employment-api",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3EmailLinkToAcas",
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "CRUD"
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:13158]: R permission

File: definitions/json/CaseEventToFields.json [hunk 5]
```diff
@@ -5484,6 +5484,169 @@
     "RetainHiddenValue": "Yes",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "CaseEventID": "et3Notification",
+    "CaseFieldID": "et3NotificationDocCollection",
+    "DisplayContext": "MANDATORY",
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields.json:5491]: This should be COMPLEX - the fields part of the complexType should then be listed on EventToComplexTypes

**Review by jsinghm13 — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 6]
```diff
@@ -13241,6 +13241,396 @@
...
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3EmailLinkToAcas",
+    "UserRole": "caseworker-employment-englandwales",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et3EmailLinkToAcas",
+    "UserRole": "caseworker-employment",
+    "CRUD": "CRUD"
```
Comment (Itay-Kirsh) [inline definitions/json/AuthorisationCaseField.json:13536]: Shouldn't be CRUD

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

File: definitions/json/AuthorisationCaseField.json (hunk 6 above)
Comment (jsinghm13) [reply definitions/json/AuthorisationCaseField.json:13536]: Updated to read only

**Review by Itay-Kirsh — APPROVED** [review]

**Review by jsinghm13 — APPROVED** [review]

**Pull request merged**